Jenkinsfile

```diff
@@ -356,6 +354,41 @@ pipeline {
               }
             }
           }
+          stage('Release') {
+            options {
+              skipDefaultCheckout()
+              timeout(time: 12, unit: 'HOURS')
```

Comment:
We wanna last the input approval for 12 hours, to let enough time for the input approval to be accepted or rejected